app/controllers/playlists_controller.rb

```diff
@@ -0,0 +1,81 @@
+class PlaylistsController < ApplicationController
+
+  def index
+
+  end
+
+  def new
+    @playlist = Playlist.new
+
+  end
+
+  def show
+    @playlist = Playlist.find(params[:id])
+    @client = Grooveshark::Client.new({session: session[:groove_session]})
+    @picture = @playlist.art_url
+    unless @picture
+      @picture = "http://1.bp.blogspot.com/-NFIeRN1TNpU/Ukou19njwHI/AAAAAAAAARQ/iypdhkQVZvI/s200/7313935-heavy-metal-rock-and-roll-devil-horns-hand-sign-with-a-black-leather-studded-bracelet.jpg"
+    end
+  end
+
+  def create
+    @playlist = Playlist.create(title: params[:playlist][:title], user_id: current_user.id, art_url: params[:playlist][:art_url])
+
+    redirect_to @playlist
+
+  end
+
+  def destroy
```

Comment:
If you haven't implemented a method, don't include a blank one or a route that goes to it.